src/app/shared/countdown-button.component.html

```diff
@@ -0,0 +1,8 @@
+<button #button type="button"
+  class="btn btn-outline-primary link-with-icon d-flex justify-content-center justify-content-sm-start"
+  [ngClass]="{'mr-3': layout.gtxs$ | async, 'mt-2': !first, 'mt-sm-0': true}"
```

Comment:
I'd suggest removing the `'mt-2': !first` clause and, externally, the code that uses would pass `<countdown-button [ngClass]="{'mr-3': layout.gtxs$ | async, 'mt-2': !first, 'mt-sm-0': true}">`